parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -904,6 +914,9 @@ impl orml_xtokens::Config for Runtime {
     type BaseXcmWeight = UnitWeightCost;
     type LocationInverter = <XcmConfig as Config>::LocationInverter;
     type MaxAssetsForTransfer = MaxAssetsForTransfer;
+    type MinXcmFee = ParachainMinFee;
+    type MultiLocationsFilter = Everything;
```

Comment:
Let's change this to `[Kusama, Karura, Moonriver]`?

Reply:
I also opened up an issue to additionally filter by asset: https://github.com/open-web3-stack/open-runtime-module-library/issues/724

Reply:
@sander2 are these locations already defined somewhere?

Reply:
> @sander2 are these locations already defined somewhere?

No not yet, but should be something like this:
- Karura: `MultiLocation::new(1, X1(Parachain(2000))`
- Moonriver: `MultiLocation::new(1, X1(Parachain(2023))`
- Kusama: `MultiLocation::new(1, Here)`

Reply:
@sander2 added please check, not sure if we should add an entry with `parents: 0`. See: https://github.com/open-web3-stack/open-runtime-module-library/blob/7e2f985d8ed49fe0d243e7ec8eb864cd9cc91d2a/xtokens/src/mock/para_relative_view.rs#L329

Reply:
No that doesn't make sense, we should never use xtokens to send something to our own chain